### PR TITLE
EC-3737 Save device data to AWS RDS for PostgresSQL using the Data Integrations

### PR DESCRIPTION
**Pull request by lucas-emqx**

### CODE REVIEW

File: en_US/rule_engine/rule_engine_save_rds_mysql.md [hunk 1]
```diff
@@ -1,134 +1,131 @@
 # Save device data to RDS for MySQL using the Data Integrations
 
-In this article, we will simulate the temperature and humidity data, and report these data to EMQX Cloud via the MQTT protocol, and then we will use EMQX Cloud Data Integrations to dump the data to RDS for MySQL.
+In this article, we will simulate temperature and humidity data and report these data to EMQX Cloud via the MQTT protocol, and then use the EMQX Cloud Data Integrations to dump the data into PostgreSQL.
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_save_rds_mysql.md:3]: It's also

**Review by zibuyu1995 — CHANGES_REQUESTED** [review]

File: en_US/rule_engine/rule_engine_save_mysql.md [hunk 2]
```diff
@@ -1,35 +1,36 @@
 # Save device data to MySQL using the Data Integrations
 
-In this article, we will simulate the temperature and humidity data, and report these data to EMQX Cloud via the MQTT protocol, and then we will use EMQX Cloud Data Integrations to dump the data to MySQL.
+In this article, we will simulate temperature and humidity data and report these data to EMQX Cloud via the MQTT protocol, and then use the EMQX Cloud Data Integrations to dump the data into PostgreSQL.
```
[the remark this answers is not in the document]
Comment (lucas-emqx) [reply en_US/rule_engine/rule_engine_save_mysql.md]: Thank you, fixed!

File: en_US/rule_engine/rule_engine_save_rds_mysql.md (hunk 1 above)
Comment (lucas-emqx) [reply en_US/rule_engine/rule_engine_save_rds_mysql.md]: Fixed!

File: en_US/rule_engine/rule_engine_save_rds_psql [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+
+    # Import the repository signing key:
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+
+    # Update the package lists:
+    sudo apt-get update
+
+    # Install the latest version of PostgreSQL.
+    sudo apt-get -y install postgresql
```
Comment (zibuyu1995) [inline en_US/rule_engine/rule_engine_save_rds_psql:55]: You only need to install the PostgreSQL client
```
sudo apt install -y postgresql-client
```

**Review by zibuyu1995 — CHANGES_REQUESTED** [review]

File: en_US/rule_engine/rule_engine_save_rds_psql (hunk 3 above)
Comment (lucas-emqx) [reply en_US/rule_engine/rule_engine_save_rds_psql]: Updated as per your request

**Review by zibuyu1995 — APPROVED** [review]

**Review by luoyq0629 — APPROVED** [review]

**Review by chenx-work — APPROVED** [review]

**Pull request merged**